lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderEntityProducer.java

```diff
@@ -2339,7 +2251,19 @@ private String transferAttachmentFiles(String msgBody, String oldSiteId, String
                         oldReferenceId = StringUtils.replace(oldReferenceId, StringUtils.replace(serverURL, "https://", "http://"), StringUtils.EMPTY);
                         oldReferenceId = StringUtils.replace(oldReferenceId, StringUtils.replace(serverURL, "http://", "https://"), StringUtils.EMPTY);
                         String newReferenceId = StringUtils.replace(oldReferenceId, oldSiteId, newSiteId);
-                        contentHostingService.copy(oldReferenceId, newReferenceId);
```

Comment:
question for @mpellicer: what was the reason for this code? I'm just wondering if we are duplicating the code in Resources

Reply:
it seems like the original intent was around attachments (not Resources) ... the original JIRA from SAK-42800 talks about an audio attachment ....